src/EnergyPlus/CMakeLists.txt

```diff
@@ -965,15 +967,24 @@ if(LINK_WITH_PYTHON)
   add_custom_command(
     TARGET energyplusapi
     POST_BUILD # TODO: I don't think we want to quote the generator expression
-    COMMAND ${Python_EXECUTABLE} "${PROJECT_SOURCE_DIR}/cmake/PythonCopyStandardLib.py" "$<TARGET_FILE:energyplusapi>" "python_standard_lib")
+    COMMAND ${Python_EXECUTABLE} "${PROJECT_SOURCE_DIR}/cmake/PythonCopyStandardLib.py" "$<TARGET_FILE:energyplusapi>" "python_lib"
+    COMMAND ${CMAKE_COMMAND} -E env --unset=PIP_REQUIRE_VIRTUALENV
+            ${Python_EXECUTABLE} -m pip install --target="$<TARGET_FILE_DIR:energyplusapi>/python_lib" --upgrade energyplus-launch==3.7.2
+  )
 endif()
 
 if(BUILD_PACKAGE)
   # if we are building package, we need to drop in some API/Plugin stuff
   if(LINK_WITH_PYTHON)
     # we'll want to grab the standard lib for python plugins
     # TODO: I don't think we want to quote the generator expression
-    install(DIRECTORY "$<TARGET_FILE_DIR:energyplus>/python_standard_lib/" DESTINATION "./python_standard_lib")
+    install(DIRECTORY "$<TARGET_FILE_DIR:energyplus>/python_lib/" DESTINATION "./python_lib")
+	if(WIN32)
+	  # on Windows, with Build Package, and also Link With Python, we can also drop in shortcuts to the new auxiliary CLI
+	  # NOTE: The install command COMPONENTS should line up so that they are run at the same packaging step
+      install(CODE "execute_process(COMMAND powershell.exe -ExecutionPolicy Bypass -File ${PROJECT_SOURCE_DIR}/scripts/dev/create_shortcut.ps1 -TargetPath \"$<TARGET_FILE:energyplus>\" -ShortcutPath \"$<TARGET_FILE_DIR:energyplus>/EPLaunchPython.lnk\" -Arguments \"auxiliary eplaunch\")" COMPONENT Auxiliary)
+	  install(FILES $<TARGET_FILE_DIR:energyplus>/EPLaunchPython.lnk DESTINATION "./" COMPONENT Auxiliary)
+	endif()
```

Comment:
An interesting choice to call a powershell script to create it in the the installed package.

I suppose this could be helpful even in the tar.gz installer.

I would have expected a modification or an addition around here for the Start Menu though, no?

https://github.com/NREL/EnergyPlus/blob/48aa15be77a0c23be01286e4576d3247932a5b10/cmake/qtifw/install_win_createstartmenu.qs#L50

Reply:
Similarly, we may want an x-energyplus.xml and a eplaunch.desktop file for debian?

and adjusting or adding to https://github.com/NREL/EnergyPlus/blob/48aa15be77a0c23be01286e4576d3247932a5b10/cmake/qtifw/install_registerfiletype.qs#L52:L54

x-energyplus.xml would probably be something like

```
<?xml version="1.0" encoding="UTF-8"?>
<mime-info xmlns="http://www.freedesktop.org/standards/shared-mime-info">

  <mime-type type="application/x-energyplus">
    <sub-class-of type="text/plain"/>
    <comment>EnergyPlus Input Data File</comment>
    <comment xml:lang="fr">Fichier d'entrées EnergyPlus</comment>
    <glob pattern="*.idf" />
  </mime-type>

 //// deal with imf and epg

</mime-info>
```

Reply:
Yeah, launchers for Mac/Linux would be nice, but I think we'll leave that for another day.  Let's let the interested parties run it on Windows and get things polished up before throwing out too many launchers. :)